misc/git-hook/necessary/precommit/check/lineendings.bash

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+. $GITDIR/hooks/functions.bash
+
+changed_files=`getChangedFiles`
+issue_found=0
+
+IFS=$'\n' # bash specific
+for file in $changed_files
+do
+	result=`cat "$file" | grep -E -n '[[:space:]]$'`
+
+	if [ -n "$result" ]
+	then
+		echo "file with line ending issue: $file"
+		echo "$result"
```

Comment:
Might be wise to not echo this. As large files tend to clutter the output a lot

Reply:
See next comment.